src/pages/index.tsx

```diff
@@ -1,17 +1,15 @@
 import type { NextPage } from 'next';
 
-import HomeMasthead from '../components/Home/HomeMasthead';
-import LatestPosts from '../components/Home/LatestPosts';
-import AboutMe from '../components/Home/AboutMe';
-import ContactMe from '../components/Home/ContactMe';
+import HomeMasthead from '../components/HomeMasthead';
+import LatestPosts from '../components/Home/LatestPosts/index';
+import Contact from '../components/Home/ContactMe'
```

Comment:
The about me section you are removing here is the one we want to keep, that is the one I merged in.

Reply:
Added